pulumi/README.md

```diff
@@ -0,0 +1,77 @@
+# Pulumi
+
+Author: [Nick Santos](https://github.com/nicks)
+
+Use [Pulumi](https://www.pulumi.com/) to deploy services to a cluster.
+
+Use this to add a fast dev loop to your Pulumi-based projects.
+
+## Requirements:
+
+- Tilt v0.23.4+
+- Pulumi
```

Comment:
+ Python3
+ macOS/Linux or WSL (for the wrapper script)

Reply:
? why wouldn't Windows work?

Reply:
oh, nm, i understand -- added a comment.